<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="listsTab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Chans">
        <attribute name="title">
         <string>Chans</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="ChansList">
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">2</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Friends">
        <attribute name="title">
         <string>Friends</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeWidget" name="FriendsList">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="convsTab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Convs">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionCreate_account"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_LiveJaming"/>
   </widget>
   <widget class="QMenu" name="menuChans">
    <property name="title">
     <string>Conversation</string>
    </property>
    <addaction name="actionNew_Chan"/>
    <addaction name="actionSearch_Chan"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Room"/>
    <addaction name="actionSearch_Room"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Friend"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChans"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionCreate_account">
   <property name="text">
    <string>Create account</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_LiveJaming">
   <property name="text">
    <string>About LiveJaming</string>
   </property>
  </action>
  <action name="actionSearch_Chan">
   <property name="text">
    <string>Search Chan</string>
   </property>
  </action>
  <action name="actionNew_Chan">
   <property name="text">
    <string>New Chan</string>
   </property>
  </action>
  <action name="actionNew_Room">
   <property name="text">
    <string>New Room</string>
   </property>
  </action>
  <action name="actionSearch_Room">
   <property name="text">
    <string>Search Room</string>
   </property>
   <property name="toolTip">
    <string>Search Room</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionAdd_Friend">
   <property name="text">
    <string>Add Friend</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
